src/main/java/uk/gov/pay/adminusers/resources/InviteResource.java

```diff
@@ -117,9 +117,8 @@ public Response completeInvite(@Parameter(example = "d02jddeib0lqpsir28fbskg9v0r
 
         return inviteServiceFactory.inviteCompleteRouter().routeComplete(inviteCode)
                 .map(inviteCompleter -> {
-                    return inviteCompleter.withData(inviteCompleteRequestFrom(payload)).complete(inviteCode)
-                            .map(inviteCompleteResponse -> Response.status(OK).entity(inviteCompleteResponse).build())
-                            .orElseGet(() -> Response.status(NOT_FOUND).build());
+                    InviteCompleteResponse inviteCompleteResponse = inviteCompleter.withData(inviteCompleteRequestFrom(payload)).complete(inviteCode);
+                    return Response.status(OK).entity(inviteCompleteResponse).build();
```

Comment:
Bonus cleanup -- DropWizard allows you to set the entity that should be sent from a successful call to the resource as the method return type, in my view this makes the code cleaner and I think it means the OpenAPI spec has more to work with etc.

This might look something like: 

```java
public InviteCompleteResponse completeInvite() {
  // ...
  return inviteServiceFactory.inviteCompleteRouter().routeComplete(inviteCode)
    .map(inviteCompleter -> inviteCompleter.withData(inviteCompleteRequestFrom(payload)).complete(inviteCode))
    .orElseThrow(() -> new WebApplicationException(Response.Status.NOT_FOUND))
}
```